node/sc/local_backend.go

```diff
@@ -76,22 +76,27 @@ func (lb *LocalBackend) BalanceAt(ctx context.Context, account common.Address, b
 		return nil, errBlockNumberUnsupported
 	}
 	statedb, _ := lb.subbrige.blockchain.State()
-	return statedb.GetBalance(account), nil
+	select {
+	case <-ctx.Done():
+		return nil, ctx.Err()
+	default:
+		return statedb.GetBalance(account), nil
+	}
 }
 
-func (lb *LocalBackend) CallContract(ctx context.Context, call klaytn.CallMsg, blockNumber *big.Int) ([]byte, error) {
+func (lb *LocalBackend) CallContract(_dummyCtx context.Context, call klaytn.CallMsg, blockNumber *big.Int) ([]byte, error) {
 	if blockNumber != nil && blockNumber.Cmp(lb.subbrige.blockchain.CurrentBlock().Number()) != 0 {
 		return nil, errBlockNumberUnsupported
 	}
 	currentState, err := lb.subbrige.blockchain.State()
 	if err != nil {
 		return nil, err
 	}
-	rval, _, _, err := lb.callContract(ctx, call, lb.subbrige.blockchain.CurrentBlock(), currentState)
+	rval, _, _, err := lb.callContract(call, lb.subbrige.blockchain.CurrentBlock(), currentState)
```

Comment:
Is it okay that we neglect caller's context and generate a new context inside of `callContract`?